dabs.sh

```diff
@@ -4,4 +4,5 @@ docker run -it --rm \
   -e USE_UID="$(id -u "${USER}")" \
   -e USE_GID="$(id -g "${USER}")" \
   -v "$(pwd)":/abs/workdir/ \
+  --network host \
```

Comment:
This is probably required for connecting to kind or other clusters.

A better solution would be to let users specify the docker network to connect to (in case of kind) or let them enable this through a `--insecure-use-host-network` flag.

For reference: https://medium.com/nttlabs/dont-use-host-network-namespace-f548aeeef575

Reply:
In general, I agree, but in this case I still think it's valid. The point is that we might want to connect to any external cluster, not only kind. And the only reasonable user experience is that the in-container (dabs) tool can connect to the same hosts as primary net ns. This means we have to reuse it. I don't think it's a security risk, because:
- we only run a temporal container
- we don't run as root, so no net configuration can be done from container anyway.

The whole point is actually to reuse primary net ns's routing config.